tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_reducers_test.ts

```diff
@@ -507,4 +512,65 @@ describe('Debugger reducers', () => {
     const nextState = reducers(state, actions.executionScrollRight());
     expect(nextState.executions.scrollBeginIndex).toEqual(1450);
   });
+
+  it(`Updates states on executionDigestFocused`, () => {
+    const state = createDebuggerState({
+      activeRunId: '__default_debugger_run__',
```

Comment:
Is this override used in any meaningful way? If not, do not override it. Instead, override what can impact the test like `executions.scrollBeginIndex`.

Reply:
Done.